kani-driver/src/assess/metadata.rs

```diff
@@ -27,6 +27,12 @@ use super::AssessArgs;
 /// This is not a stable interface.
 #[derive(Serialize, Deserialize)]
 pub struct AssessMetadata {
+    // TODO: The assess cache should check this field to know whether the version should change.
```

Comment:
Assess scan has an `--existing-only` option that won't rerun assess but only parse existing metadata files. What I'm trying to state here, is that this option shouldn't reuse metadata files that were generated by older versions of `Kani`, since the results might no longer hold.

I can rewrite this comment to make it clearer.